data/json/npcs/random_encounters/camper_van_traveler.json

```
@@ -0,0 +1,381 @@
+[
+  {
+    "type": "npc",
+    "id": "traveler_camper_van",
+    "class": "NC_SCAVENGER",
+    "attitude": 0,
+    "mission": 3,
+    "chat": "TALK_NPC_CAMPER_VAN_TRAVELER",
+    "faction": "no_faction"
+  },
+  {
+    "type": "npc_class",
+    "id": "NC_CAMPER_VAN_TRAVELER",
+    "name": { "str": "Scavenger" },
+    "job_description": "I'm just trying to survive.",
+    "traits": [ { "group": "BG_survival_story_EVACUEE" }, { "group": "NPC_starting_traits" }, { "group": "Appearance_demographics" } ],
+    "shopkeeper_price_rules": [
+      { "category": "food", "premium": 2.5 },
+      { "item": "gasoline", "premium": 2.5 },
+      { "item": "water_clean", "premium": 2.5 }
+    ],
+    "skills": [
+      { "skill": "ALL", "level": { "sum": [ { "dice": [ 3, 2 ] }, { "constant": -3 } ] } },
+      { "skill": "gun", "bonus": { "rng": [ 2, 4 ] } },
+      { "skill": "pistol", "bonus": { "rng": [ 2, 5 ] } },
+      { "skill": "rifle", "bonus": { "rng": [ 0, 3 ] } },
+      { "skill": "archery", "bonus": { "rng": [ 0, 3 ] } }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": [
+      "TALK_NPC_CAMPER_VAN_TRAVELER",
+      "TALK_NPC_CAMPER_VAN_TRAVELER_DESTINATION",
+      "TALK_NPC_CAMPER_VAN_TRAVELER_BACKSTORY",
+      "TALK_NPC_CAMPER_VAN_TRAVELER_LEAVING_WHEN"
+    ],
+    "responses": [
+      {
+        "text": "Do you have anything that you'd like to sell?",
+        "topic": "TALK_NPC_CAMPER_VAN_TRAVELER_TRADE",
+        "condition": { "npc_has_var": "knows_u", "type": "dialogue", "context": "first_meeting", "value": "yes" }
+      },
+      {
+        "text": "Where did you say you were going again?",
+        "topic": "TALK_NPC_CAMPER_VAN_TRAVELER_DESTINATION",
+        "condition": { "npc_has_var": "knows_u", "type": "dialogue", "context": "first_meeting", "value": "yes" }
+      },
+      {
+        "text": "What did you do before <the_cataclysm>?",
+        "topic": "TALK_NPC_CAMPER_VAN_TRAVELER_BACKSTORY",
+        "condition": { "npc_has_var": "knows_u", "type": "dialogue", "context": "first_meeting", "value": "yes" }
+      },
+      {
+        "text": "Have you found anything out of the ordinary out in the wastes?",
+        "topic": "TALK_NPC_CAMPER_VAN_TRAVELER_INTERESTING_THINGS",
+        "condition": {
+          "and": [
+            { "npc_has_var": "knows_u", "type": "dialogue", "context": "first_meeting", "value": "yes" },
+            {
+              "not": { "npc_has_var": "gave_directions", "type": "dialogue", "context": "camper_van_traveler", "value": "yes" }
+            }
+          ]
+        },
+        "effect": [
+          {
+            "arithmetic": [ { "npc_val": "var", "var_name": "direction", "type": "randomize", "context": "dialogue" }, "=", { "rand": 4 } ]
+          },
```

```suggestion
          { "math": [ "npc_randomize_dialogue_direction", "=", "rand(4)" ] },
```
I'm currently replacing this arithmetic comparisons with math, but it's WIP